components/social_links.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import twitterIcon from '../images/icons/twitter.svg';
+import githubIcon from '../images/icons/github.svg';
+import dribbbleIcon from '../images/icons/dribbble.svg';
+
+const Icons = {
+  twitter: twitterIcon,
+  github: githubIcon,
+  dribbble: dribbbleIcon,
+};
+
+export default class SocialLinks extends React.Component {
```

Comment:
It just strikes me as odd if when you do

```javascript
import React, { PropTypes } from 'react';
```

but then neglect the `Component`, instead explicitly using the `React` namespace. I get that it's more explicit. But the context in which this component is being written is already explicit enough, IMO. Even so that the first reaction I drew from this was "oh, this is inconsistent".

But if you have a strong opinion regarding it, then go ahead.